### DISCUSSION TITLE
When is DivestOS going to update to Android 12?

### DISCUSSION THREAD

**Discussion by MarkCyberSecDev93** [Divested-Mobile/DivestOS-Build#35] [Q&A] [answered]
Is it as fast as most normal custom ROMS? Or Is it as fast as LineageOS when it comes to this update?

**Comment by SkewedZeppelin** [comment, marked as answer]
Please look at the history page:

https://divestos.org/index.php?page=history

Earliest March, latest May.
I am in no rush.
